pkg/ddc/juicefs/operations/base.go

```diff
@@ -319,17 +319,13 @@ func (j JuiceFileUtils) exec(command []string, verbose bool) (stdout string, std
 }
 
 // execWithoutTimeout
-func (j JuiceFileUtils) execWithoutTimeout(command []string, verbose bool) (stdout string, stderr string, err error) {
+func (j JuiceFileUtils) execWithoutTimeout(command []string) (stdout string, stderr string, err error) {
 	stdout, stderr, err = kubeclient.ExecCommandInContainer(j.podName, j.container, j.namespace, command)
 	if err != nil {
-		j.log.Info("Stdout", "Command", command, "Stdout", stdout)
+		j.log.V(1).Info("Stdout", "Command", command, "Stdout", stdout)
```

Comment:
I think it's not user friendly when debugging issue.

Reply:
`info` directly ?